### PR TITLE
Supplement isort options

### PR DESCRIPTION
**Pull request by moreal**
This pull request is related to the https://github.com/charliermarsh/ruff/issues/1749 issue but it doesn't resolve the problem perfectly yet. I opened this pull request to check the intermediate stage with reviewing.

I referenced https://github.com/PyCQA/isort/blob/3a72e069635a865a92b8a0273aa829f630cbcd6f/isort/settings.py source file.

### CODE REVIEW

File: crates/ruff/src/flake8_to_ruff/isort.rs [hunk 1]
```diff
@@ -1,10 +1,160 @@
...
+    pub add_imports: HashSet<String>,
+    #[serde(default)]
+    pub remove_imports: HashSet<String>,
+    #[serde(default)]
+    pub append_only: bool,
+    #[serde(default)]
+    pub reverse_relative: bool,
+    #[serde(default)]
+    pub force_single_line: bool,
+    #[serde(default)]
```
Comment (charliermarsh) [inline crates/ruff/src/flake8_to_ruff/isort.rs:156]: These may all need the `alias = "single-line-exclusions", alias = "single_line_exclusions"` treatment as seen in `src_paths`, to respect isort's acceptance of either casing style.

**Comment by charliermarsh** [comment]
Seems like a reasonable start! I might suggest only implementing those options that we support within Ruff for now, to make things easier. see: https://beta.ruff.rs/docs/settings/#isort.

File: crates/ruff/src/flake8_to_ruff/isort.rs (hunk 1 above)
Comment (moreal) [reply crates/ruff/src/flake8_to_ruff/isort.rs]: I added such aliases too! Thanks! 🙏🏻

**Comment by moreal** [comment]
> Seems like a reasonable start! I might suggest only implementing those options that we support within Ruff for now, to make things easier. see: https://beta.ruff.rs/docs/settings/#isort.

Thank you for your suggestion. I added more options from https://beta.ruff.rs/docs/settings/#isort and clean up other options, not in the document. 🙏🏻

**Comment by charliermarsh** [comment]
I'm going to close this for now, just to keep the open PR list tight. If you choose to return to this, we can always reopen!

**Pull request closed without merging**